docs/Basic-Guide.md

```diff
@@ -27,21 +25,36 @@ for more information.
        Equivalent or .NET 4.x Equivalent)**
 6. Go to **File** > **Save Project**
 
+## Setting up the Inference Engine
+
+In order to run pre-trained models of agents behaviors, you will need to set-up the 
```

Comment:
We provide pre-trained models for all the agents in our all demo environments. To be able to run those models, you'll first need to set-up the Inference Engine.